bin/elastic-version

```diff
@@ -6,20 +6,27 @@
 # The version can also be forced by setting the ELASTIC_VERSION environment variable.
 
 import json
-from os import environ
+import os
 
 
 def get_hard_coded_version():
     version_info = json.load(open('version.json'))
     return version_info['version']
 
 
+def qualify(version):
+    qualifier = os.getenv('BEAT_VERSION_QUALIFIER')
```

Comment:
Does it make sense for `apm-server-docker` to get it's qualifier from an environment variable named `BEAT_VERSION_QUALIFIER`?

Maybe it could use `VERSION_QUALIFIER` like `ml-cpp`?
https://github.com/elastic/release-manager/blob/master/build.gradle#L208

Reply:
yeah, let's do it.